dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/FFDCData.java

```diff
@@ -23,43 +23,50 @@ public FFDCData() {
-                                           LogFieldConstants.STACKTRACE,
-                                           LogFieldConstants.OBJECTDETAILS,
-                                           LogFieldConstants.SEQUENCE
-    };
-
-    public static final String[] NAMES1_1 = {
-                                              LogFieldConstants.IBM_DATETIME,
+    public static final String[] NAMES_LC = {
+                                              LogFieldConstants.DATETIME,
                                               LogFieldConstants.DATEOFFIRSTOCCURENCE,
                                               LogFieldConstants.COUNT,
                                               LogFieldConstants.MESSAGE,
-                                              LogFieldConstants.IBM_CLASSNAME,
+                                              LogFieldConstants.CLASSNAME,
                                               LogFieldConstants.LABEL,
-                                              LogFieldConstants.IBM_EXCEPTIONNAME,
-                                              LogFieldConstants.IBM_PROBEID,
+                                              LogFieldConstants.EXCEPTIONNAME,
+                                              LogFieldConstants.PROBEID,
                                               LogFieldConstants.SOURCEID,
-                                              LogFieldConstants.IBM_THREADID, //long
-                                              LogFieldConstants.IBM_STACKTRACE,
-                                              LogFieldConstants.IBM_OBJECTDETAILS,
-                                              LogFieldConstants.IBM_SEQUENCE,
-                                              LogFieldConstants.HOST,
-                                              LogFieldConstants.IBM_USERDIR,
-                                              LogFieldConstants.IBM_SERVERNAME,
+                                              LogFieldConstants.THREADID,
+                                              LogFieldConstants.STACKTRACE,
+                                              LogFieldConstants.OBJECTDETAILS,
+                                              LogFieldConstants.SEQUENCE,
+                                              LogFieldConstants.HOSTNAME,
+                                              LogFieldConstants.WLPUSERDIR,
+                                              LogFieldConstants.SERVERNAME,
                                               LogFieldConstants.TYPE
     };
 
-    private static NameAliases jsonLoggingNameAliases = new NameAliases(NAMES1_1);
+    public static final String[] NAMES_JSON = {
+                                                LogFieldConstants.IBM_DATETIME,
+                                                LogFieldConstants.DATEOFFIRSTOCCURENCE,
+                                                LogFieldConstants.COUNT,
+                                                LogFieldConstants.MESSAGE,
```

Comment:
`dateOfFirstOccurrence` and `count` don't seem to be used, but `message` is. From an FFDC event generated on a clean wlp for 20.0.0.7:

```
{"type":"liberty_ffdc","host":"192.168.64.1","ibm_userDir":"\/Users\/jennifer.zhen.chengibm.com\/20007\/wlp\/usr\/","ibm_serverName":"defaultServer","ibm_datetime":"2020-07-28T09:49:03.205-0400","message":"PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed","ibm_className":"com.ibm.ws.ssl.core.WSX509TrustManager","ibm_exceptionName":"sun.security.validator.ValidatorException","ibm_probeID":"checkServerTrusted","ibm_threadId":"00000032","ibm_stackTrace":"[snipped for space]","ibm_objectDetails":"[snipped for space]","ibm_sequence":"1595944143205_0000000000001"}
```

Is this concerning? 😓

Reply:
no, that's exactly the right list of fields to see in an ffdc JSON.  `message` is a special case that we didn't add an `ibm_` prefix to because it makes it fit better with other events that you might want to show in the same views.

So, that said, I'm wondering if we should block out the field names we don't use (eg. with a constant like "XXXXXXX" rather than fill in the array with invalid (but harder to notice) ones?